test/integration/other/select-many.js

```diff
@@ -87,15 +88,17 @@ describe('select many value processing', () => {
     const asAlice = await service.login('alice');
 
     // the select1 version of forms.selectMultple
-    const selectOne = `<?xml version="1.0"?>
+    const selectOne = palatableXML(`<?xml version="1.0"?>
       <h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa">
         <h:head>
           <model>
             <instance>
               <data id="selectMultiple">
                 <meta><instanceID/></meta>
                 <q1/>
-                <g1><q2/></g1>
+                <g1>
+                  <q2/>
+                </g1>
```

Comment:
Is this a white space change only? If so, I think it'd be ideal to do that in a separate PR.